web/turkey/survey/static/survey/js/auditors/visibility_changes.js

```diff
@@ -0,0 +1,17 @@
+// helper script file for focus_change auditors
```

Comment:
No way to load helper script in current framework... I suppose I could add option for multiple loaded scripts. My elimination of duplicates would do the rest. But even that is problematic because we would need a dependency resolution order for scripts. So I think we either have a global helper script for all auditors or auditor js files need to stand on their own.